config.json

```diff
@@ -114,6 +114,11 @@
                         "price": {}
                     }
                 },
+                "giveaway": {
+                    "Commands": {
+                        "view": {}
```

Comment:
Added this as I'd forgotten when I implemented the feature. It's required so that the "admin" module can be used to disable the command, in case the server administrator would like to limit its usage to specific channels.